html-aam/roles-contextual.html

```diff
@@ -45,6 +45,15 @@
 <section data-testname="el-section" aria-label="x" data-expectedrole="region" class="ex">x</section>
 <section data-testname="el-section-no-name" class="ex-generic">x</section>
 
+<!-- el-summary -->
+<summary data-testname="el-summary-orphan" class="ex-generic">x</summary>
+<details>
+  <ol>
+    <li>x</li>
+    <li>x</li>
+  </ol>
+  <summary data-testname="el-summary-not-first-child-of-details" class="ex-generic">x</summary>
```

Comment:
this wouldn't be a generic, because it's the first summary that is a child of the details element.

so 
```
<details>
  whatever...
  <summary> this is not a generic </summary>
  <summary> this is a generic </summary>
</details>
```

Reply:
Fixed (I had mixed up the requirement for `<summary>` to be the first child of `<details>` which throws a parsing error).